backend-go/internal/core/hello/handler.go

```
@@ -16,15 +17,20 @@ type Handler struct {
+	if err := req.Validate(); err != nil {
+		return middleware.NewError(http.StatusBadRequest, "Invalid JSON", err)
 	}
+
 	id, err := h.service.createHello(req)
 	if err != nil {
-		return http.NewErrorResponse(c, err)
+		return middleware.NewError(http.StatusInternalServerError, "server internal error", err)
 	}
-	return http.NewOkResponse(c, models.HelloResponse{
+
+	return c.JSON(http.StatusOK, models.HelloResponse{
```

golangci-lint (1.62.2)

error returned from interface method should be wrapped: sig: func (github.com/labstack/echo/v4.Context).JSON(code int, i interface{}) error

(wrapcheck)